client/my-sites/people/delete-user/with-delete-user.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import page from 'page';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import useDeleteUserMutation from 'calypso/data/users/use-delete-user-mutation';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+
+const useSuccessNotice = ( isSuccess, user, isMultisite, siteSlug ) => {
+	const showNotice = React.useRef();
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	React.useEffect( () => {
+		showNotice.current = () => {
+			dispatch(
+				successNotice(
+					isMultisite
+						? translate( 'Successfully removed @%(user)s', {
+								args: { user: user.login },
+								context: 'Success message after a user has been modified.',
+						  } )
+						: translate( 'Successfully deleted @%(user)s', {
+								args: { user: user.login },
+								context: 'Success message after a user has been modified.',
```

Comment:
It's a bit vague we use `modified` and not `deleted` or `removed` in these contexts. But it's pre-existing, so 🤷

Reply:
I agree, although it doesn't affect the translated string. As far as I can see `context` is used for the initial translation of the string, changing it would trigger a new translation request which is why I left it like this (see [here](https://github.com/Automattic/wp-calypso/pull/50386/files/f99312643556e128cba927eef011617b5f86ac74#diff-235d573700276e58d9d91ef3179cd15a1e4bd569b111c8c4fa17b1e38dc2cd97)).